### DISCUSSION TITLE
CHR 95.13% all the time

### DISCUSSION THREAD

**Discussion by kora24** [i3/i3#6026] [Q&A]
Hello,

My "i3status.conf" has these lines:
`battery all {
        format = "%status %percentage %remaining"
        path = "/sys/class/power_supply/BAT%d/uevent"
}
`

The status bar shows "CHR 95.13%" _all the time_.

I have run
`cat /sys/class/power_supply/BAT0/uevent`
and it shows the following:
`POWER_SUPPLY_NAME=BAT0
POWER_SUPPLY_TYPE=Battery
POWER_SUPPLY_STATUS=Charging
POWER_SUPPLY_PRESENT=1
POWER_SUPPLY_TECHNOLOGY=Li-ion
POWER_SUPPLY_CYCLE_COUNT=12
POWER_SUPPLY_VOLTAGE_MIN_DESIGN=11250000
POWER_SUPPLY_VOLTAGE_NOW=12843000
POWER_SUPPLY_CURRENT_NOW=0
POWER_SUPPLY_CHARGE_FULL_DESIGN=3735000
POWER_SUPPLY_CHARGE_FULL=3553000
POWER_SUPPLY_CHARGE_NOW=3553000
POWER_SUPPLY_CAPACITY=100
POWER_SUPPLY_CAPACITY_LEVEL=Normal
POWER_SUPPLY_MODEL_NAME=L20D3PG
POWER_SUPPLY_MANUFACTURER=Sunwoda
POWER_SUPPLY_SERIAL_NUMBER=BD93`

What should I change in the "i3status.conf" in order to see the real charge in the status bar?

Thank you.

**Comment by okraits** [comment]
You could use `battery 0 { ...}`instead. Take a look at the manpage.

**Comment by kora24** [comment]
I have changed `battery all` to `battery 0` but still it does not show 100%.

**Comment by CyrilSLi** [comment]
Maybe check out [this question](https://github.com/i3/i3/discussions/6032)?
